backend/fms_core/services/property_value.py

```diff
@@ -14,9 +14,11 @@ def create_properties_from_values_and_types(properties, property_types_objs_dict
 
         if type(value).__name__ in ('datetime', 'time'):
             value = value.isoformat().replace("T00:00:00", "")
+        else:
+            value = str(value) if value else ' '
```

Comment:
To be verified but I believe that Pandas ingest all the cells as string. Therefore I'm not quite sure if type(value).__name__ would ever return 'datetime' and 'time'
Also if it is already a string, no need to str(value) in the second part